### PR TITLE
Allow the use of URL parameters when creating a paginator.

### PR DESCRIPTION
**Pull request by autresphere**
This PR cancels and replaces previous PR #2039

### CODE REVIEW

**Comment by autresphere** [comment]
Please tell me if you need more tests on this feature.

File: Tests/Logic/ObjectMapping/RKPaginatorTest.m [hunk 1]
```diff
@@ -474,6 +482,18 @@ - (void) testLoadingAPageWithCustomPaginator
...
+- (void) testLoadingAPageWithCustomPaginatorContainingParameter
+{
+    RKObjectManager* manager = [RKTestFactory objectManager];
+    [RKObjectManager setSharedManager:manager];
+    
+    manager.paginationMapping = [self customPaginationMapping];
+    RKPaginator* paginator = [manager paginatorWithPathPattern:RKPaginatorTestResourcePathPattern parameters:@{@"param1":@"value1"}];
+    [paginator loadPage:1];
+    [paginator waitUntilFinished];
+    expect(paginator.URL.relativeString).to.contain(@"param1=value1");
```
Comment (segiddins) [inline Tests/Logic/ObjectMapping/RKPaginatorTest.m]: could you also just make sure loading a second page preserves the parameters? Thanks!

**Comment by autresphere** [comment]
As requested another test has been added. Do you need more test coverage?

**Comment by segiddins** [comment]
Thanks @autresphere !

**Pull request merged**